src/ngraph/frontend/onnx_import/op/topk.cpp

```diff
@@ -79,6 +92,31 @@ namespace ngraph
                 }
             }
 
+            namespace set_11
+            {
+                NodeVector topk(const Node& node)
+                {
+                    // Process inputs
+                    auto data = node.get_ng_inputs().at(0);
+                    auto k = get_k(node);
+
+                    // Process attributes
+                    const auto axis = get_axis(node);
+                    const auto largest = node.get_attribute_value<std::int64_t>("largest", 1);
+                    const auto sorted = node.get_attribute_value<std::int64_t>("sorted", 1);
```

Comment:
You can read these attributes to `bool` variable.

example from `gemm` operator:
```
                    bool trans_a = node.get_attribute_value<int64_t>("transA", 0);
                    bool trans_b = node.get_attribute_value<int64_t>("transB", 0);
```

Reply:
I could, but we're changing this line in the `v1` upgrade this sprint.